Cargo.toml

```diff
@@ -13,9 +13,9 @@ categories = ["no-std", "embedded"]
 exclude = [ "demo/*" ]
 
 [dependencies]
-pwasm-alloc = { path = "alloc", version = "0.3" }
+pwasm-alloc = { path = "alloc", version = "0.4" }
 pwasm-libc = { path = "libc", version = "0.2" }
-tiny-keccak = "1.3"
+tiny-keccak = "1"
```

Comment:
seems unrelated to the pr?

Reply:
This overall PR is about updating dependencies and code to make it work with latest compiler. So it's soundly to make this change here for at least two reasons:

1. It's how all other dependencies are written, so it's more consistent with overall library behavior
1. There is no reason to not use latest version if it doesn't break the code
1. Project is currently using version 1.4 (you can check Cargo.lock) so it was misleading and confusing to have here one version one another one is actually is in use.